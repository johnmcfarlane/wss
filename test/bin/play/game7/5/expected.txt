 48  3,9  | PAH
 48  3,9  | PEH
 43  3,9  | HAP
 43  3,9  | HEP
 42  3,9  | PEA
 39  1,9  - HAPI
 39  1,9  - HOPI
 36  1,9  - PAHI
 36  2,10 - PALEA
 34  3,9  | HAE
 34  3,9  | HAO
 30  1,5  - HEP
 30  1,5  - HOP
 27  2,5  - OP
 27  1,5  - PEH
 27  1,5  - POH
 26  3,5  | HOWEA
 25  5,5  - EPHA
 25  5,5  - HOPE
 25  6,8  - PEHO
 24  3,9  | AAH
 24  6,8  - APH
 24  5,5  - HEP
 24  5,5  - HOP
 24  1,9  - PEAI
 24  6,8  - PEH
 23  2,5  - EH
 23  2,8  - HEXA
 23  2,5  - OH
 22  3,10 - ALEPH
 22  3,10 - ALPHA
 22  5,5  - HEAP
 22  6,5  - OPAH
 21  3,5  | HOWE
 21  6,8  - OPAH
 21  3,9  | PA
 21  3,9  | PE
 20  1,8  - APEX
 20  5,5  - EPA
 20  6,5  - EPHA
 20  7,8  - EPHA
 20  2,10 - HALP
 20  2,10 - HELP
 20  2,10 - HOLP
 20  5,5  - HP
 20  6,5  - PAHA
 20  6,5  - PAHO
 20  6,5  - PHO
 20  7,8  - PHO
 20  6,5  - POHA
 19  6,5  - HOPE
 19  6,5  - HP
 19  6,5  - PH
 19  7,8  - PH
 18  3,9  | AAA
 18  6,6  | ADAPA
 18  6,6  | ADPAO
 18  3,10 - ALAPA
 18  6,8  - APA
 18  6,8  - APE
 18  2,8  - HEX
 18  7,5  - HOPE
 18  3,10 - OLPAE
 18  6,8  - OPA
 18  6,8  - OPE
 18  6,5  - PAH
 18  7,5  - PAHA
 18  7,5  - PAHO
 18  1,10 - PAOLA
 18  6,8  - PEA
 18  7,5  - PEHO
 18  6,5  - POH
 18  7,5  - POHA
 17  7,8  - EPA
 17  3,9  | HA
 17  6,5  - HAP
 17  7,5  - HAP
 17  3,9  | HE
 17  7,5  - HEP
 17  6,5  - HOP
 17  7,5  - HOP
 17  7,5  - PAH
 17  7,8  - PAHA
 17  7,8  - PAHO
 17  7,5  - PEH
 17  7,8  - PEHO
 17  7,5  - POH
 17  7,8  - POHA
 16  3,10 - ALAP
 16  3,10 - ALOHA
 16  3,10 - ALOP
 16  3,10 - ELOAH
 16  6,5  - EPA
 16  1,10 - HAOLE
 16  3,9  - HIPE
 16  4,10 - LEAP
 16  4,10 - LEPA
 16  4,10 - LOPE
 16  3,10 - OLPE
 16  6,5  - OPA
 16  1,10 - OPAL
 16  6,5  - OPE
 16  1,10 - PAAL
 16  7,8  - PAH
 16  2,10 - PALA
 16  2,10 - PALE
 16  1,10 - PEAL
 16  7,8  - PEH
 16  3,10 - PLEA
 16  6,5  - POA
 16  6,5  - POE
 16  7,8  - POH
 16  2,10 - POLE
 15  3,10 | AH
 15  3,10 | EH
 15  3,8  - EX
 15  7,5  - HEAP
 15  3,9  - HIP
 15  6,5  - OP
 15  6,8  - OP
 15  3,6  | OWE
 15  6,5  - PA
 15  6,8  - PE
 15  7,5  - PHO
 15  6,5  - PO
 14  7,5  - AHEAP
 14  7,5  - APH
 14  5,5  - EH
 14  6,5  - HAE
 14  2,10 - HALA
 14  2,10 - HALE
 14  2,10 - HALO
 14  6,5  - HAO
 14  5,5  - HE
 14  1,10 - HEAL
 14  5,5  - HO
 14  6,5  - HOE
 14  2,10 - HOLA
 14  2,10 - HOLE
 14  4,10 - LAP
 14  4,10 - LEAH
 14  4,10 - LEP
 14  4,10 - LOP
 14  2,10 - PAL
 14  7,8  - PEA
 14  7,8  - POA
 14  7,8  - POE
 14  2,10 - POL
 13  6,5  - EH
 13  7,8  - EH
 13  6,5  - HA
 13  6,5  - HO
 13  7,3  | HOPES
 13  6,5  - OH
 13  7,5  - OPAH
 13  7,8  - PA
 13  7,8  - PE
 13  7,5  - PEA
 13  7,5  | PESAH
 13  7,5  - PH
 13  7,8  - PO
 13  7,5  - POA
 13  7,5  - POE
 13  3,5  | POW
 12  2,10 - HAL
 12  2,10 - HEL
 12  2,10 - HOL
 12  7,5  - HP
 12  4,9  - IPH
 12  4,10 - LAH
 11  7,5  - AAH
 11  7,5  - HAE
 11  7,5  - HAO
 11  7,5  - HOE
 11  3,5  | HOW
 11  4,9  - IHP
 11  7,5  - PA
 11  7,5  - PE
 11  2,9  - PHI
 11  3,9  - PIA
 11  3,9  - PIE
 11  7,5  - PO
 10  3,10 - ALAE
 10  3,10 - ALEA
 10  3,10 - ALOE
 10  7,5  - APA
 10  7,5  - APE
 10  5,5  - EO
 10  2,9  - EPI
 10  2,9  - OHIA
 10  3,10 - OLEA
 10  7,5  - OPA
 10  7,5  - OPE
 10  1,5  - PE
 10  1,5  - PO
  9  3,9  | AA
  9  3,10 | AA
  9  6,8  - AE
  9  3,9  | AE
  9  3,10 | AE
  9  7,5  - AHA
  9  7,5  - AHO
  9  6,5  - EO
  9  3,10 | EO
  9  7,5  - HA
  9  2,3  | HAAF
  9  1,5  - HE
  9  7,5  - HE
  9  2,3  | HEAF
  9  3,9  - HIA
  9  3,9  - HIE
  9  1,5  - HO
  9  7,5  - HO
  9  6,8  - OE
  9  3,9  | OE
  9  3,9  - PI
  8  2,10 - AAL
  8  6,3  | AHEAD
  8  2,9  - AHI
  8  2,4  | HAF
  8  4,10 - LAO
  8  4,10 - LEA
  8  4,10 - LEO
  8  4,10 - LOA
  8  4,10 - LOE
  8  7,5  - OP
  8  5,10 | OP
  8  5,4  | PEEL
  7  7,5  - AAA
  7  6,6  | ADP
  7  1,5  - AE
  7  7,5  - AH
  7  5,10 | AH
  7  3,10 - ALP
  7  7,4  | APES
  7  1,5  - EO
  7  7,8  - EO
  7  6,5  | HADE
  7  6,4  | HEAD
  7  5,4  | HEEL
  7  3,9  - HI
  7  1,5  - OE
  7  7,5  - OH
  7  5,10 | OH
  7  3,10 - OLP
  7  7,4  | OPES
  7  6,5  | PAD
  7  7,4  | PEES
  7  7,5  | PESA
  7  7,5  | PESO
  7  2,9  - POI
  6  7,6  | ESP
  6  6,5  | HAD
  6  7,4  | HAES
  6  2,9  - HEI
  6  5,5  | HEL
  6  7,4  | HOES
  6  2,9  - HOI
  6  2,4  | OAF
  6  7,5  | PES
  5  7,5  - AA
  5  5,10 | AA
  5  7,5  - AE
  5  5,10 | AE
  5  2,5  | EF
  5  7,5  | HES
  5  7,5  - OE
  5  5,10 | OE
  5  2,5  | OF
  4  6,6  | ADA
  4  6,6  | ADO
  4  3,10 - ALA
  4  3,10 - ALE
  4  6,5  | EAD
  4  5,5  | EEL
  4  3,10 - ELA
  4  4,9  - IAO
  4  6,5  | OAD
  4  3,10 - OLA
  4  3,10 - OLE
  3  7,5  | AES
  3  3,9  - AI
  3  3,10 - AL
  3  3,10 - EL
  3  7,6  | ESE
  3  4,9  - IO
  3  4,10 - LA
  3  4,10 - LO
  3  7,5  | OES
  3  3,9  - OI
