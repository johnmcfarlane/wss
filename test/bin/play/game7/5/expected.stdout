2,8:PAH=48
2,8:PEH=48
2,8:HAP=43
2,8:HEP=43
2,8:PEA=42
1,9:PALEA=36
2,8:HAE=34
2,8:HAO=34
0,4:HEP=30
0,4:HOP=30
1,4:OP=27
0,4:PEH=27
0,4:POH=27
4,4:EPHA=25
2,8:AAH=24
1,4:EH=23
1,4:OH=23
2,9:ALEPH=22
2,9:ALPHA=22
2,4:HOWE=21
2,8:PA=21
0,7:APEX=20
1,9:HELP=20
1,9:HOLP=20
5,4:HOPE=19
1,7:HEX=18
6,4:HOPE=18
5,4:POH=18
2,8:HA=17
6,4:HAP=17
2,8:HE=17
6,4:HEP=17
5,4:HOP=17
6,4:HOP=17
6,4:PAH=17
6,4:PEH=17
6,4:POH=17
2,9:ALOHA=16
0,9:HAOLE=16
3,9:LEAP=16
3,9:LOPE=16
0,9:OPAL=16
1,9:PALE=16
0,9:PEAL=16
2,9:PLEA=16
1,9:POLE=16
2,9:AH=15
2,9:EH=15
2,7:EX=15
6,4:HEAP=15
2,8:HIP=15
2,5:OWE=15
4,4:EH=14
1,9:HALE=14
1,9:HALO=14
0,9:HEAL=14
1,9:HELO=14
5,4:HOE=14
1,9:HOLE=14
3,9:LAP=14
3,9:LOP=14
1,9:PAL=14
1,9:POL=14
5,4:HO=13
6,2:HOPES=13
6,4:OPAH=13
6,4:PEA=13
2,4:POW=13
6,4:HAE=11
6,4:HAO=11
6,4:HOE=11
2,4:HOW=11
6,4:PA=11
1,8:PHI=11
2,8:PIA=11
2,8:PIE=11
2,9:ALAE=10
2,9:ALOE=10
1,8:OHIA=10
2,9:OLEA=10
6,4:OPE=10
2,8:AA=9
2,9:AA=9
2,8:AE=9
2,9:AE=9
6,4:HA=9
1,2:HAAF=9
0,4:HE=9
6,4:HE=9
2,8:HIE=9
0,4:HO=9
6,4:HO=9
2,8:OE=9
2,8:PI=9
1,9:AAL=8
5,2:AHEAD=8
1,8:AHI=8
3,9:LEA=8
4,9:OP=8
6,4:OP=8
4,3:PEEL=8
0,4:AE=7
4,9:AH=7
2,9:ALP=7
6,3:APES=7
5,4:HADE=7
5,3:HEAD=7
4,3:HEEL=7
2,8:HI=7
0,4:OE=7
4,9:OH=7
6,4:OH=7
6,3:OPES=7
5,4:PAD=7
6,3:PEES=7
6,4:PESO=7
1,8:POI=7
5,4:HAD=6
6,3:HAES=6
6,3:HOES=6
1,3:OAF=6
6,4:PES=6
4,9:AA=5
4,9:AE=5
1,4:EF=5
6,4:HES=5
6,4:OE=5
4,9:OE=5
1,4:OF=5
5,5:ADO=4
2,9:ALA=4
2,9:ALE=4
4,4:EEL=4
2,9:OLE=4
2,8:AI=3
2,9:AL=3
2,9:EL=3
3,9:LA=3
3,9:LO=3
6,4:OES=3
2,8:OI=3
